src/Appwrite/Utopia/Response/Model/Document.php

```diff
@@ -36,6 +36,12 @@ public function __construct()
                 'default' => '',
                 'example' => '5e5ea5c16897e',
             ])
+            ->addRule('$sequence', [
+                'type' => self::TYPE_INTEGER,
+                'description' => 'Document automatically incrementing ID.',
```

Comment:
We might want to have a small section for this on the docs. Let's discuss with DevRel.